practice_06.08.2019_System.IO/Task1_Folders/Folder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace practice_06._08._2019_System.IO.Task1_Folders
+{
+    public class Folder
+    {
+        public void ShowFoldersAndSize()
+        {
+            var path = @"C:\Windows\System32";
+            DirectoryInfo directory = new DirectoryInfo(path);
+            int foldersNumber = directory.GetDirectories().Length;
+
+            double size = 0;
```

Comment:
Why do we initialize size with zero and then rewrite it?
Please use return value from method for initialization.